code-scanning/psalm.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Psalm Security Scan
+
+on:
+  push:
+    branches: [ $default-branch, $protected-branches ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ $default-branch ]
+  schedule:
+    - cron: $cron-weekly
+
+permissions:
+  contents: read
+
+jobs:
+  php-security:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Psalm Security Scan
+        uses: psalm/psalm-github-security-scan@f3e6fd9432bc3e44aec078572677ce9d2ef9c287
```

Comment:
I don't see any updated on this repo since few years and only 4 releases. You are confident that it is something we need? And this org is not even verified

Reply:
We regularly advise users who wants to scan their PHP codebase to use Psalm and it yield decent results. [Psalm](https://github.com/vimeo/psalm) itself is still actively maintainer.
The action itself could get a bit of TLC. Let me see what could be done about it.

Reply:
This action still has to be part of our [technology partner program](https://partner.github.com/apply?partnershipType=Technology+Partner) or be part of github or actions org.

Reply:
Hi @Phantsure! I am planning to take contact with them in general to inform them about this initiative and also ask them if they wish to be part of [technology partner program](https://partner.github.com/apply?partnershipType=Technology+Partner)

Reply:
Great!

Reply:
This [issue](https://github.com/psalm/psalm-github-security-scan/issues/6) has been raised to open a discussion on this topic